ntfy/cli.py

```diff
@@ -49,9 +49,10 @@ def send_notification(message, args, config):
         module = import_module('ntfy.backends.{}'.format(backend))
 
         try:
+            backend_config = config.get(backend, {})
+            backend_config.update(args.option)
```

Comment:
Make an Issue :wink:

I'm not opposed, I'd just want it done in a backwards compatible way preferably (tho that should be completely possible)